Sources/Basic/OutputByteStream.swift

```diff
@@ -55,32 +58,108 @@ public class OutputByteStream: OutputStream {
         return buffer.count
     }
 
-    /// The contents of the output stream.
-    ///
-    /// This method implicitly flushes the stream.
-    public var bytes: ByteString {
-        flush()
-        return ByteString(self.buffer)
+    /// Currently available buffer size.
+    private var availableBufferSize: Int {
+        return max(buffer.capacity, OutputByteStream.bufferSize) - buffer.count
```

Comment:
It is defensive though, I like this being in terms of `buffer.capacity`. Jinx on the other stuff :)